test/skate_web/channels/vehicles_channel_test.exs

```diff
@@ -73,12 +35,25 @@ defmodule SkateWeb.VehiclesChannelTest do
 
     test "subscribes to vehicles for a run ID", %{socket: socket} do
       assert {:ok, %{data: []}, %Socket{} = socket} =
-               subscribe_and_join(socket, VehiclesChannel, "vehicles:run_ids:123-4567")
+               subscribe_and_join(socket, VehiclesChannel, "vehicles:run_ids:" <> @vehicle.run_id)
+    end
+
+    test "subscribes to vehicles for a block ID", %{socket: socket} do
```

Comment:
if we're going to support multiple block IDs in this call I'd like to see a test case for that

Reply:
I added a test to the channel and also a test to `Realtime.Server` (which tests the actual logic more directly).